CHANGES.md

```diff
@@ -12,6 +12,12 @@
   1. Generator API `xcube.core.gen2.LocalCubeGenerator` and
     `xcube.core.gen2.service.RemoteCubeGenerator`;
   2. Generator CLI `xcube gen2`.
+* xcube now issues a warning, if a data cube is opened from object 
+  storage, and credentials have been neither passed not can be found, 
```

Comment:
```suggestion
  storage, and credentials either were passed or can not be found, 
```

Reply:
not --> nor is correct

Reply:
I f you want to use ''nor', say:
"credentials neither were passed nor can they be found" .
I don't like this solution as it sounds weird, so I still stand by my suggestion (I just noticed it is missing a "not").